libs/store-harvester/src/store/encrypted.ts

```
@@ -0,0 +1,77 @@
+  //   const suffix = this.getDbSuffix();
+  //   return path.join(this.config.rootFolder, db_name ?? `config${suffix}.db`);
+  // }
+
+
+  protected async loadDb(): Promise<PouchDB.Database<Stored>> {
+    try {
+      const db = new PouchDB<Stored>(this.dbPath, { adapter: 'memory' });
+      await this.initEncryptedDb(db, this.config.password);
+      return db;
+    } catch (err) {
+      const isLockError = err instanceof Error &&
```

ESLint

[error] '&&' should be placed at the beginning of the line.

(operator-linebreak)